apps/nextjs/src/stores/AilaStoresProvider.tsx

```diff
@@ -78,14 +79,29 @@ export const AilaStoresProvider: React.FC<AilaStoresProviderProps> = ({
     haveInitialized.current = true;
   }, [stores.lessonPlan, id, stores.moderation, stores.chat]);
 
+  useEffect(() => {
```

Comment:
I'm not familiar with our state management implementation and Zustand, but does this useEffect want a dependency array? Or is the whole point that it's called every render and the control flow is then handled within the `subscribe()` callback?

Reply:
Oh wow, why didn't this give a lint warning. You're completely right, thank you

Reply:
Fixed